Update DG - manual testing (#205)
AY2122S2-CS2103T-T11-3/tp: docs/DeveloperGuide.md

       Expected output:  error message detailing what went wrong
   
2. Feature with multiple parameters
    
   The following examples are tested on a list containing 1 or more applications and is targeted towards testing the edit command,
the steps are general enough to be used to test other commands that accept multiple parameters.
   
   
   2. Test valid input values
      - E.G. `edit INDEX [Optional parameters]` <br>
        Test input: `edit 1 n/new name` when list has more than 1 application <br>
        Expected output: Successful change of application 1 name
   3. Test invalid input values
      - E.G.  `edit INDEX [Optional parameters]` <br>
        Test input: `edit 0 n/new name` or `edit -10 n/new name` <br>
        Expected output: error message detailing what went wrong
   4. Test invalid command input
       - E.G.  `edit INDEX [Optional parameters]` <br>
         Test input: `edi 1 n/new name` or `edit 2 2 n/new name` <br>
         Expected output: error message detailing what went wrong
   5. Test multiple parameter combinations
       - E.G. `edit INDEX [optional parameters]` <br>
         Test input: `edit 1 n/new name n/actual name` <br>

>          Expected output: error message detailing what went wrong
>          Actual output: command executes

Reviewer: ```suggestion
         Expected output: command executes with the last parameter
```
Is it supposed to have both "Expected output" and "Actual output"?

Author: yes, I wrote it from the testers point of view, since typing in multiple would have an unexpected output for our application. We might want to discuss on this, since I wasn't too sure when writing this part

</div>
### Example: Deleting an application

1. Deleting an application while all applications are being shown

   1. Prerequisites: List all applications using the `list` command. Multiple applications in the list.

   1. Test case: `delete 1`<br>
      Expected: First application is deleted from the list. Details of the deleted application shown in the status message.

   1. Test case: `delete 0`<br>
      Expected: No application is deleted. Error details shown in the status message. Status bar remains the same.

   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
      Expected: Similar to previous.

2. Deleting an application while reminder window is open
   1. Prerequisites: Have the reminder window open using `reminder` and 
   set the first applications interview date to be within the next 7 days.

   2. Test case: `delete 1`<br>
      Expected: First application is deleted from the list. Details of the deleted application shown in the status message.
      Application is also deleted from reminder window

   3. Test case: `delete 0`<br>

>       Expected: No application is deleted. Error details shown in the status message. Status bar remains the same.

Reviewer: Did you mean the summary bar remains the same?

Author: I was following the previous examples, where they referenced a status bar, assuming it refers to the IDE status bar
